include/seastar/core/abort_source.hh

```diff
@@ -140,9 +149,6 @@ public:
                   std::is_nothrow_invocable_r_v<void, Func>)
     [[nodiscard]]
     optimized_optional<subscription> subscribe(Func&& f) {
-        if (abort_requested()) {
-            return { };
-        }
         if constexpr (std::is_invocable_v<Func, std::exception_ptr>) {
```

Comment:
Isn't the documentation no longer correct?

Reply:
Good catch. I'll address that in the next version.

Reply:
fixed in v2